File: Basic Java/Basic/Array.java
package com.cdac.iet;/*Write an array and display its elements*/


import java.util.Scanner;


public class Array
{
	public static void main(String args[])
{
	int []arr =new int[5];
	System.out.println("Enter 5 elements of array");
	Scanner sc=new Scanner(System.in);
	for(int i=0;i<arr.length;i++)
	{
	 arr[i]=sc.nextInt();
	}
	System.out.println("Array is:");	
 	for(int i=0;i<arr.length;i++)
	{
	System.out.printf("\t%d",arr[i]);
	}
}

}

File: C Programs/Assignment 4/array.c
/*write a program to get array*/


#include<stdio.h>
int main()
{
	int arr[5],i;
	printf("Enter elements of array");
	for(i=0;i<=4;i++)
	{
		scanf("%d",&arr[i]);
	}
	printf("\n Entered elements are..");
	for(i=0;i<=4;i++)
	{
		printf("\t%d",arr[i]);
	}
}
